server/index.js

```diff
@@ -1,9 +1,42 @@
 require('dotenv/config');
+const pg = require('pg');
 const express = require('express');
+const ClientError = require('./client-error');
 const staticMiddleware = require('./static-middleware');
+const uploadsMiddleware = require('./uploads-middleware');
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL
+});
 
 const app = express();
 
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
+
+app.post('/api/post-form', uploadsMiddleware, (req, res, next) => {
+  const { title, tags, content } = req.body;
+  const tagsArray = JSON.stringify([tags]);
```

Comment:
You are going to need to do something besides put them in an array. If you look at your database data, you'll notice that more than one tag is double-wrapped.

_After_ your validation. You are going to need to write some logic to determine if `tags` is already an array. See MDN for `Array.isArray`.